config.yaml

```diff
@@ -22,5 +22,7 @@ tft:
           - name: measure_cpu
           - name: measure_power
           - name: validate_offload
+        # Secondary network is required for tests 27, 28 and 29
```

Comment:
@thom311 `POD_TO_POD_2ND_INTERFACE_SAME_NODE`, `POD_TO_POD_2ND_INTERFACE_DIFF_NODE` and `POD_TO_POD_MULTI_NETWORK_POLICY` are of type `TestCaseType` which is one level below the `TestType`. So this check I believe cannot be applied here. 
I have raised a ValueError in case `secondary_network_nad `is None in `get_secondary_ip()` since this method gets called only for these three TestCaseTypes.
```
def get_secondary_ip(self) -> str:
        if self.ts.connection.secondary_network_nad == None:
            raise ValueError(f'Secondary Network is missing')
```

Do you think this is good enough?

Reply:
why is `namespace` argument being added to the `parse `method for this fix?

Reply:
Thanks for the suggestion! I modified it accordingly